docs/how-to-use-relational-backend-storage.md

```diff
@@ -6,8 +6,8 @@ license: "This software is licensed under the Apache License version 2."
 
 ## Introduction
 
-Before the version `0.5.0`, Apache Gravitino only supports KV backend storage to store metadata. Since
-RDBMS is widely used in the industry, starting from the version `0.5.0`, Gravitino supports using
+Before the version `0.6.0`, Apache Gravitino supports KV and Relational backend storage to store metadata. Since KV storage is
+hard to manage and RDBMS is widely used in the industry, starting from the version `0.6.0`, Gravitino will not support KV backend, and it only supports using
```

Comment:
You don't need to explain the reason.

"Since KV storage is
hard to manage and RDBMS is widely used in the industry, starting from the version `0.6.0`, Gravitino will not support KV backend, and it only supports using"

=>

"Since `0.6.0`, Gravitino only supports using"